code/modules/mod/mod_theme.dm

```diff
@@ -932,16 +932,16 @@
 	wound = 5
 
 /datum/mod_theme/syndicate
-	name = "syndicate"
-	desc = "A suit designed by Gorlex Marauders, offering armor ruled illegal in most of Spinward Stellar."
+	name = "keres"
```

Comment:
the weird skin names can stay but if not syndicate, then why not name it `suspicious` or `tactical` or something

Reply:
why would a corporation/private military/terrorist group name their own thing "suspicious" afaik, the modsuit names aren't OOC things so like. besides, they're all about ancient greek deities, when the skins become actual suits it'll fit better

Reply:
> why would a corporation/private military/terrorist group name their own thing "suspicious" afaik, the modsuit names aren't OOC things so like. besides, they're all about ancient greek deities, when the skins become actual suits it'll fit better

yeah and you can change your skin from keres but the suit still stays named the same. i think tactical/combat would be fine?

Reply:
ohhhhhhhhhh

ok fair hm